hack/images/xslttransform/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM debian:11
+
+RUN apt update && apt install -y libxml2
```

```suggestion
RUN apt update && apt install -y --no-install-recommends libxml2 \
 && rm -rf /var/lib/apt/lists/*
```

`--no-install-recommends` disables optional dependencies from being installed. However for the packages installed, i don't see any optional dependencies. I added it just in case a future version has some optional dependencies.

`rm -rf /var/lib/apt/lists/*` clears up apt listing..

After applying these changes, i see that the image on disk went from 178MB to 117MB

---

done